package.json

```diff
@@ -46,9 +46,10 @@
     "cozy-app-publish": "^0.27.2",
     "cozy-bar": "7.16.0",
     "cozy-client": "22.1.0",
-    "cozy-device-helper": "1.12.0",
+    "cozy-device-helper": "1.16.0",
     "cozy-doctypes": "1.72.2",
     "cozy-flags": "2.6.0",
+    "cozy-intent": "^1.3.0",
```

Comment:
should we use fixed dependencies or should we use ^ ?

the PR does not look consistent in my opinion.

Reply:
Fixed